code/modules/mob/living/bot/ed209bot.dm

```diff
@@ -30,7 +32,7 @@
 
 	new /obj/item/weapon/secbot_assembly/ed209_assembly(Tsec)
 
-	var/obj/item/weapon/gun/energy/taser/G = new /obj/item/weapon/gun/energy/taser(Tsec)
+	var/obj/item/weapon/gun/energy/taser/G = new used_weapon(Tsec)
```

Comment:
Not sure if this will cause problems?

Reply:
I'm not sure either, so... We'll find out!